kong/conf_loader/init.lua

```diff
@@ -1053,6 +1053,16 @@ local function check_and_infer(conf, opts)
       errors[#errors + 1] = "only in-memory storage can be used when role = \"data_plane\"\n" ..
                             "Hint: set database = off in your kong.conf"
     end
+
+    if not conf.lua_ssl_trusted_certificate then
+      conf.lua_ssl_trusted_certificate = {}
+    end
+
+    if conf.cluster_mtls == "shared" then
+      table.insert(conf.lua_ssl_trusted_certificate, conf.cluster_cert)
```

Comment:
We can check to see if `conf.cluster_cert` and `conf.cluster_ca_cert` are already inside `conf.lua_ssl_trusted_certificate` here, in case the user specified it manually using `lua_ssl_trusted_certificate`.

Reply:
Discussed with @fffonion and this should not affect OpenSSL when loading. So mark this as resolved unless it became a problem later.